server/assets/javascripts/reset-password.js

```diff
@@ -10,6 +10,7 @@ document.getElementById('email').placeholder = '';
         this.addEventListener('load', function () {
             const isForSendCode = requestURL.indexOf('SendCode') > 0;
             const isForVerifyCode = requestURL.indexOf('VerifyCode') > 0;
+            const isForVerifyMFA = requestURL.indexOf('Verify') > 0;
```

Comment:
This will be true when `isForVerifyCode` is true, this probably isn't what we want.